src/components/Common/Avatar.tsx

```diff
@@ -66,15 +66,15 @@ const Avatar = React.forwardRef<
         <AvatarPrimitive.Image
           src={imageUrl}
           alt={name}
-          className="aspect-square h-full w-full object-cover"
+          className="aspect-square object-cover"
         />
       ) : (
         <AvatarPrimitive.Fallback className="flex h-full w-full select-none items-center justify-center text-center">
           <svg
             xmlns="http://www.w3.org/2000/svg"
             version="1.1"
             viewBox="0 0 100 100"
-            className="aspect-square h-full w-full object-cover"
+            className="aspect-square object-cover"
```

Comment:
here too. i don't think anything in this file needs to be modified in order to achieve what you want.